site/docs/mark/boxplot.md

```diff
@@ -0,0 +1,136 @@
+
+
+2) __`min-max` Box Plot__, which is a box plot where lower and upper whiskers are defined as the min and max respectively.
+
+<div class="vl-example" data-name="boxplot_minmax_2D_horizontal"></div>
+
+## Dimension & Orientation
+There are two `boxplot` dimensions:
+
+1) A 1D `boxplot` shows the distribution of a continuous field.
+<div class="vl-example" data-name="boxplot_tukey_1D_horizontal"></div>
+
+2) A 2D `boxplot` shows the distribution of a continuous field, broken down by categories.
+<div class="vl-example" data-name="boxplot_tukey_2D_horizontal_IQR"></div>
+
+Box plot's orientation is automatically determined by the continuous field axis.
+For example, you can create a vertical 1D box plot by encoding a continuous field on the y axis.
+
+<div class="vl-example" data-name="boxplot_tukey_1D_vertical"></div>
+
+For 2D box plots with one continuous and one discrete fields,
+the box plot will be horizontal if the continuous field is on the x axis.
+
+<div class="vl-example" data-name="boxplot_tukey_2D_horizontal_IQR"></div>
+
+Similarly, if the continuous field is on the y axis, the box plot will be vertical.
+
+<div class="vl-example" data-name="boxplot_tukey_2D_vertical"></div>
+
+## Color, Size, and Opacity Encoding Channels
+
+You can customize the color, size, and opacity of the box in the `boxplot` by using the `color`, `size`, and `opacity` [encoding channels](encoding.html#channels). `color` and `size` are applied to only the box and median tick whereas `opacity` is applied to the whole `boxplot`.
+
+An example of a `boxplot` where the `size` encoding channel is specified.
+<div class="vl-example" data-name="boxplot_vertical_2D_vertical"></div>
+
+<div class="vl-example" data-name="boxplot_minmax_2D_horizontal_color_size"></div>
+
+## Role Config
+
+To customize different parts of the box, we can use roles config to customize different parts of the box plot (`box`, `median`, `whisker`).
+
+{: .suppress-error}
+```json
+{
+  "config": {
+    "box": ...,
+    "median": ...,
+    "whisker": ...
+  }
+}
+```
+<div class="vl-example" data-name="boxplot_minmax_2D_horizontal_custom_midtick_color"></div>
+
+These are possible because under the hood, the `"boxplot"` mark is a macro that expands into a layered plot.  For example, [a basic 1D boxplot shown above](#boxplot-type) is expanded to:
+
+<div class="vl-example" data-name="normalized/boxplot_tukey_1D_horizontal"></div>
+
+__Note:__ `aggregate` property of the continuous field is implicitly `boxplot`.
+For example, [a basic 1D boxplot shown above](#boxplot-type) is equivalent to:
+<div class="vl-example" data-name="boxplot_tukey_2D_vertical_explicit_aggregate"></div>
+
+## Mark Config
+{: .suppress-error}
+```json
+{
+  "boxplot": {
+    "size": ...,
+    "extent": ...,
+    "box": ...,
+    "median": ...,
+    "whisker": ...
+  }
+}
+```
+The `boxplot` config object sets the default properties for `boxplot` composite marks.
```

Comment:
This explanation is incomplete.    Link to mark properties like we do in the master branch?